config/routes.rb

```diff
@@ -13,6 +13,8 @@
       end
       resources :progresses, only: :index
       resources :blogs, only: [:show, :index]
+      resources :facilitator_guides, only: :show
+      resources :lesson_notes, only: :show
```

Comment:
@andela-mpitan Remember what we did for `facilitor_guides`. Lesson notes should be nested under `sessions`. After doing that let us remove line 16 and 17 since they would no longer be needed